### PR TITLE
Updated tutorial cats-effect dependency to 3.0.0

### PR DESCRIPTION
**Pull request by lrodero**
Updated tutorial cats-effect dependency to `3.0.0`

### CODE REVIEW

**Comment by vasilmkd** [comment]
Can you please change to Scala 2.13.5 and cats-effect 3.0.1? Thank you 😊

**Comment by lrodero** [comment]
Done :slightly_smiling_face:

File: docs/tutorial.md [hunk 1]
```diff
@@ -42,11 +42,11 @@ running the code snippets in this tutorial it is recommended to use the same
 ```scala
 name := "cats-effect-tutorial"
 
-version := "3.0.0-RC2"
+version := "3.0.1"
```
Comment (vasilmkd) [inline docs/tutorial.md:45]: Are we using this anywhere? This is a comment on the file, not on this change.

Comment (vasilmkd) [reply docs/tutorial.md:45]: Oh wait, this is the version string of the sbt project?

**Review by djspiewak — APPROVED** [review]

**Review by vasilmkd — APPROVED** [review]

File: docs/tutorial.md (hunk 1 above)
Comment (lrodero) [reply docs/tutorial.md:45]: It is used in the [repo](https://github.com/lrodero/cats-effect-tutorial/tree/series/3.x) where the tutorial full code is kept. I think it is a good idea to keep the version there. Although, truly, it _should_ always be the same than the CE version being discussed in the tutorial. So I'm also fine if we want to remove it.

Comment (djspiewak) [reply docs/tutorial.md:45]: I think it's all cool to keep it :-) Makes sense to me

Comment (lrodero) [reply docs/tutorial.md:45]: Then we are cool :relaxed:

**Pull request merged**